test/multiverse/suites/ruby_kafka/Envfile

```diff
@@ -0,0 +1,14 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+suite_condition('Skip in CI on newer ruby versions') do
+  # will run locally OR on CI for ruby < 3.3.0
+  !ENV['CI'] || RUBY_VERSION < '3.3.0'
```

Comment:
```suggestion
  # will run locally OR on CI for ruby < 3.4.0
  !ENV['CI'] || RUBY_VERSION < '3.4.0'
```

I love the `suite_condition` idea. I think the range should include Ruby v3.3.0, as it exists at the time the instrumentation is being added.